Aula1-Resposta/ExercicioDivisao/index.php

```diff
@@ -0,0 +1,43 @@
+<?php 
+    $divisao = "";
+    $msgErro = "";
+    if(isset($_POST['number1']) && isset($_POST['number2']) && $_POST['number2'] != 0){
+      $number1 = $_POST['number1'];
+      $number2 = $_POST['number2'];
+      $divisao = $number1 / $number2;
+      $msgErro = "";
+    }

```

Comment:
para aparecer a mensagem em caso de erro ficaria assim, dai na linha 36 você usaria a variavei msgErro!

```suggestion
try {
    if(isset($_POST['number1']) && isset($_POST['number2'])){
      $number1 = $_POST['number1'];
      $number2 = $_POST['number2'];
      $divisao = $number1 / $number2;
    }
} catch (\Throwable $e) {
      $msgErro = $e->getMessage();
}
```